src/EditorFeatures/Core/Extensibility/Completion/CompletionListEventArgs.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.VisualStudio.Language.Intellisense;
+using System;
+
+namespace Microsoft.CodeAnalysis.Editor
+{
+    internal class CompletionListSelectedEventArgs
+    {
+        public readonly int? newValue;
```

Comment:
Null for oldValue means this is the first time a CompletionSet is getting selected. Null for newValue means that a CompletionSet provided by some other language services has been selected. Should probably document that.